GameOfLife/Board.php

```diff
@@ -17,14 +17,20 @@ class Board
      *
      * Indicates the width of the board at the end
      */
-    private $width;
+    public $width;
```

Comment:
Why is that public? Member variables should always be private, unless it is a simple object that only contains data and no methods, but that is not the case here.
Please make this private and also `$height` and `$board`.